src/screens/ProfileScreen/elements/BottomSheetBio.js

```diff
@@ -52,28 +53,42 @@ const styles = StyleSheet.create({
     fontFamily: fonts.inter[400],
     fontWeight: 'bold',
     fontSize: 24,
-    color: colors.black,
+    color: COLORS.black,
     marginBottom: dimen.normalizeDimen(12)
   },
   description: {
     fontFamily: fonts.inter[400],
     fontSize: 12,
-    color: colors.gray,
+    color: COLORS.gray,
     marginTop: 7
   },
   errorText: {
     fontFamily: fonts.inter[400],
     fontSize: 12,
-    color: colors.red,
+    color: COLORS.red,
     marginTop: dimen.normalizeDimen(7)
   },
   button: {
-    backgroundColor: colors.bondi_blue
+    marginTop: 33,
+    backgroundColor: COLORS.holyTosca
   },
   textStyling: {
     fontFamily: fonts.inter[600],
     fontSize: 18,
-    color: colors.white
+    color: COLORS.white
+  },
+  input: {
+    backgroundColor: COLORS.lightgrey,
+    paddingVertical: 16,
+    paddingHorizontal: 12,
+    height: 150,
+    justifyContent: 'flex-start',
+    overflow: 'scroll',
+    borderRadius: 8,
+    fontFamily: fonts.inter[500],
+    fontSize: 14,
+    color: COLORS.black,
+    lineHeight: 24
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color values in the styles have been updated to use constants from the theme file instead of hardcoded values. This is a good practice for maintaining consistency across the application and makes it easier to update colors in the future if needed.
```diff
- color: colors.black,
+ color: COLORS.black,

- color: colors.gray,
+ color: COLORS.gray,

- color: colors.red,
+ color: COLORS.red,

- backgroundColor: colors.bondi_blue
+ backgroundColor: COLORS.holyTosca,

- color: colors.white
+ color: COLORS.white,

- color: colors.black,
+ color: COLORS.black,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->